mypyc/irbuild/mapper.py

```diff
@@ -132,7 +132,12 @@ def fdef_to_sig(self, fdef: FuncDef) -> FuncSignature:
         else:
             # Handle unannotated functions
             arg_types = [object_rprimitive for arg in fdef.arguments]
-            ret = object_rprimitive
+            # We at least know the return type for __init__ methods will be None.
+            is_init_method = fdef.name == '__init__' and not fdef.is_module_level_func
```

Comment:
I think that you can just use `... and bool(fdef.info)`. Your suggestion to use a property is reasonable, but we have many checks where we do the equivalent of `if fdef.info` so it would add some inconsistency.

Reply:
Okay thank you that makes sense. Applying this update and the suggestions above now.